hw/misc/hermes.c

```diff
@@ -504,7 +615,7 @@ static void hermes_bar0_write(void *opaque, hwaddr addr, uint64_t val,
             return;
         }
 
-        if ((val & HERMES_CMD_START) == 1) {
+        if (val == HERMES_CMD_START) {
```

Comment:
this should have been fixed in the second commit

Reply:
This is not applicable any more, as this block of code was moved to the third commit.